core/workspace_svg.js

```diff
@@ -1539,7 +1555,7 @@ Blockly.WorkspaceSvg.prototype.pasteBlock_ = function(xmlBlock) {
           blockY += Blockly.SNAP_RADIUS * 2;
         }
       } while (collide);
-      block.moveBy(blockX, blockY);
+      block.moveTo({x: blockX, y: blockY});
```

Comment:
I think this is technically expecting a `Blockly.utils.Coordinate`. Would you mind updating to use that instead? Also, I'm a bit surprised closure isn't complaining.

Reply:
I think this works because [coordinate](https://github.com/google/blockly/blob/f94b1db85018b5a7261fa01f7d04cd5c8367fe2c/core/utils/coordinate.js#L26) is a struct. But you're right, using the constructor is more future proof.